qa/L0_backend_python/test.sh

```diff
@@ -35,23 +35,41 @@ if [ -z "$REPO_VERSION" ]; then
     exit 1
 fi
 
-DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
+# On windows the paths invoked by the script (running in WSL) must use
+# /mnt/c when needed but the paths on the tritonserver command-line
+# must be C:/ style.
+export TEST_WINDOWS=0
+if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
```

Comment:
I wonder why don't we just let the CI provide the `TEST_WINDOWS` flag, similar to how the `TEST_JETSON` is provided?

Reply:
I suppose we could do this, I was following the [previous convention](https://github.com/triton-inference-server/server/blob/main/qa/L0_sequence_batcher/test.sh#L96) from our existing Windows tests.

Reply:
I'll create a follow-up ticket for this since I believe we have plans to completely overhaul the windows testing process on the gitlab side. This overhaul is the only reason I'm hesitant to change it now, given the CI stability and the proximity to the deadline. Introducing it would also require changes to other test scripts that leave the scope of this ticket.